ui/src/components/modals/DeleteModelModal.tsx

```diff
@@ -0,0 +1,89 @@
+import { Modal, Typography } from 'antd';
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useNotification } from '../../utils/NotificationContext';
+import { useDeleteModel } from '../../hooks/models';
+import { SchemaTabs } from '../../pages/SchemaDetails';
+
+interface DeleteModelModalProps {
+  open: boolean;
+  closeModal: () => void;
+  catalog: string;
+  schema: string;
+  model: string;
+  existingVersions: boolean;
```

Comment:
can we rephrase this more like a boolean like `isXXX` or `hasXXX`
```suggestion
  hasExistingVersions: boolean;
```